### ISSUE TITLE
Update documentation for the options

### ISSUE THREAD

**Issue by lukeapage** [less/less-docs#96]
In progress (by me), but this is just to track, we are missing alot of options.

**Comment by felixfbecker** [comment]
Indeed. I can see the options for the command line, but how do they translate to the API? Are they all just camel cased?

**Comment by seven-phases-max** [comment]
>Are they all just camel cased?

Yes. (For API options it's worth to look at the most and always up-to-date and valid documentation of it, the [`lessc` script](https://github.com/less/less.js/blob/master/bin/lessc) itself).
